app/controllers/LogoutController.scala

```diff
@@ -47,26 +47,25 @@ class LogoutController @Inject()(
 
       logger.info(s"[Claiming][Session ID: ${Session.id(hc)}] user signed out from the service")
 
-      if(appConfig.logoutAudit) {
+      val auditData = Map(
+        "sessionId" -> Session.id(hc),
+        "event" -> "signout",
+        "service" -> "claim-a-trust-frontend",
+        "userGroup" -> request.affinityGroup.toString
+      )
 
-        val auditData = Map(
-          "sessionId" -> Session.id(hc),
-          "event" -> "signout",
-          "service" -> "claim-a-trust-frontend",
-          "userGroup" -> request.affinityGroup.toString
-        )
+      val auditDataWithUtr = request.userAnswers.get(IdentifierPage).fold(auditData) { identifier =>
 
-        val auditDataWithUtr = request.userAnswers.get(UtrPage).fold(auditData) { utr =>
-          auditData ++ Map("utr" -> utr)
-        }
-
-        auditConnector.sendExplicitAudit(
-          "trusts",
-          auditDataWithUtr
-        )
+        val key = if(identifier.length == 10) "utr" else "urn"
```

Comment:
Could `identifier.length == 10` be extracted to its own method as it looks to be used a few times from what I can see. Would make it easier as well if we decide to make that check a bit more rigorous

Reply:
Sure!